homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -130,15 +126,22 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
 class Remote:
     """The Remote class. It stores the TV properties and the remote control connection itself."""
 
+    _control: RemoteControl | None = None
+    state: MediaPlayerState | None = None
+    available: bool = False
+    volume: int = 0
+    muted: bool = False
+    playing: bool = True
```

Comment:
Why define these as class attributes?

Reply:
why not?

Reply:
They belong to the instance which can cause issues as instance variables don't overwrite class variables.
```py
class A:
    var: int = 2

    def func(self) -> None:
        self.var = 4
        print(f"{A.var=}")
        print(f"{self.var=}")

A().func()
```
```py
# python test.py
A.var=2
self.var=4
```

It only makes sense to define them on the class if they aren't modified as it's often the case with `_attr_xxxx` (e.g. `_attr_device_class`).